frontend/webapp/containers/overview/overview/overview.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+import React, { useCallback, useMemo, useState } from "react";
+import { KeyvalDataFlow, KeyvalLoader } from "@/design.system";
+import { QUERIES } from "@/utils/constants";
+import { useQuery } from "react-query";
+import { getDestinations, getSources } from "@/services/setup";
+import { getEdges, groupSourcesNamespace, getNodes } from "./utils";
+import { OverviewDataFlowWrapper } from "./overview.styled";
+
+export function OverviewContainer() {
+  const [containerHeight, setContainerHeight] = useState(0);
+
+  const containerRef = useCallback((node: HTMLDivElement) => {
+    if (node !== null) {
+      setContainerHeight(node.getBoundingClientRect().height);
+    }
+  }, []);
+
+  const { isLoading, data: destinations } = useQuery(
+    [QUERIES.API_DESTINATIONS],
+    getDestinations
+  );
+
+  const { data: sources } = useQuery([QUERIES.API_SOURCES], getSources);
```

Comment:
we do we track the `isLoading` for just one API call? I think we should either do it for both or none, no?